app/models/project.rb

```diff
@@ -11,6 +11,84 @@ class Project < ApplicationRecord
   has_many :project_ocean_regions
   has_many :ocean_regions, through: :project_ocean_regions
 
+  def self.filters_to_json
+    projects = Project.all.order(id: :asc)
+    unique_donors = Donor.pluck(:name).sort
+    unique_countries = Country.pluck(:name).sort
+    unique_ocean_based_regions = OceanRegion.pluck(:name).sort
+    unique_categories = projects.pluck(:category).uniq.sort
```

Comment:
I have noticed that in my database, I have some values for `category` which are `nil` so I've got an exception during sorting.
Since we should not allow empty values for those filterable columns, we should work something out.
For the sake of this PR, what we could do is get rid of those `nil` values by also doing `.compact` before `.uniq.sort`.
At a later stage, we could also add some model (and possibly database) validations.

Reply:
Ah sorry @Levia when I tested with the latest CSV I didn't seem to get an issue but maybe I didn't notice it?

Reply:
I will update this code and I will push again.